script/test/utils.sh

```diff
@@ -63,6 +63,12 @@ CONTAINERD_FLAGS+="--address ${CONTAINERD_SOCK#"unix://"} \
 
 containerd_groupid=
 
+# NOTE: We don't have the sudo command on Windows.
+sudo=""
+if [ $(id -u) -ne 0 ] && [ command -v sudo &> /dev/null ]; then
+  sudo="sudo"
```

Comment:
L105 still uses sudo.
Also, pkill does not exist in sh for windows, we should do something similar to what we did for sudo.

Reply:
Done